addons/mass_mailing/static/src/scss/mass_mailing.ui.scss

```diff
@@ -356,6 +356,14 @@ body.editor_enable.o_basic_theme.o_in_iframe {
                 overflow-y: scroll;
                 max-height: 200px;
             }
+
```

Comment:
Commit message grammar moment:
- "This commit fix" -> "This commit fixes"

Now about its content, you name the rules that are problematic but you give no information about the reason. I know we don't fully know it, but we at least can say that it's the Popper library that manipulates `transform` and we need to stop it from doing so because its calculations are all wrong.

Reply:
I see you copy-pasted from my comment directly into your code and into the commit message but I just wrote that to say _what_ you should explain ^^' This doesn't really describe the problem you fixed, doesn't address the `bottom` and `top` properties, and isn't really a full sentence :-P (also, please add a space after a `//`)